docs/content/integration/proxies/forwarded-headers/index.md

```diff
@@ -70,11 +70,18 @@ achieve a secure result. Please see the [Cloudflare] documentation on
    below).
 9. Set the `Then` section dropdown to `Remove`.
 10. Set the `Then` section `Header name` to `X-Forwarded-For`.
-11. Click `Save`.
+11. Click `Deploy`.
 
 {{< figure src="cloudflare_1.png" alt="Steps 1 to 4" width="736" >}}
```

Comment:
```suggestion
{{< figure
src="cloudflare_1.png"
alt="Image of cloudflare dashboard with buttons for steps 1 to 4 labeled."
width="736"
caption="Steps 1 - 4: Create a New Request Header Transform Rule"
title="Steps 1 - 4: Create a New Request Header Transform Rule"
>}}
```
Using the built in caption, tooltip, and improve the alt text.

Reply:
Thanks, I wasn't sure how to properly handle that in markdown.